arches/hosts.py

```diff
@@ -0,0 +1,5 @@
+from django_hosts import patterns, host
+
+host_patterns = patterns('',
+    host(r'arches', 'arches.urls', name='arches'),
```

Comment:
Could you give an example of this? This is just following the documentation for `django-hosts` https://django-hosts.readthedocs.io/en/latest/

Reply:
Haven't tested, but since these are regexes, I assume something like changing `arches` to `arches$` so that it doesn't match `arches-lingo`.

This is just the core arches hosts.py which will never have arches-lingo installed, but I figure people look to these settings as examples to riff on, and it would seem a shame if something only supposed to match `arches` matched `arches-for-science`.

Reply:
no yeah that's fair, will get the change in 👍

Reply:
undoing this as it prevented navigation to subdomains. Tested with this pattern:

```
host_patterns = patterns('',
    host(r'foo$', 'foo.urls', name='foo'),
    host(r'foo-bar$', 'bar.urls', name='foo-bar'),
)
```

the regex anchors prevented navigation to foo-bar.localhost:8000. However removing the anchors allowed proper navigation to the subdomain, even though theoretically `foo` should have matched first